ONSdigital/eq-terraform
# Env vars for author superuser

**What**

Creates the environment variables needed to create a superuser for the author application

**How to test**

1) Check out the branch
2) Create the author environment
3) Verify from the elastic beanstalk console that the environment variables have been created

**Who can test**

Anyone except @weapdiv-david
